### PR TITLE
Created welcome.mjml and Implement Welcome email

### PR DESCRIPTION
**Pull request by udobeke**
​
## Description

@udobeke: Created a welcome email template to welcome new users on sign-up
@TolulopeJoel: Implemented sending of Welcome email to users after sign-up
​
## Related Issue (Link to linear ticket)

​
## Motivation and Context

@udobeke: Welcome email templates are required because they streamline the process of creating and sending welcome emails, ensure consistency, provide a platform for personalization, and help users make a positive first impression on our app. It is a valuable tool in the overall customer engagement and retention strategy.
​
## How Has This Been Tested?
@TolulopeJoel: This change was tested by performing the following steps:

1. Created a new user account using Swagger documentation.
2. Confirmed the successful creation of the user account.
3. Received a confirmation email.
4. Verified the contents and functionality of the received email.
​
## Screenshots (if appropriate - Postman, etc):
**screenshot of mjml template (@udobeke)**
Screen Shot 2023-10-18 at 9 55 28 PM

**Screenshots of testing using swagger docs (@TolulopeJoel)**
Screenshot 2023-10-20 at 4 41 34 AM
Screenshot 2023-10-20 at 4 41 15 AM

**Screenshots of email received**
Screenshot 2023-10-20 at 5 02 34 AM

​
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
​
## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: app/services/mail_service.py [hunk 1]
```diff
@@ -87,3 +87,42 @@ def send_otp(recipient_address: str, otp: str, subject: str) -> None:
...
+
+
+def send_welcome_mail(recipient_address: str, username: str) -> None:
+    """
+    Sends a welcome email to a new user.
+
+    Parameters:
+        recipient_address (str): The email address of the recipient.
+        username (str): The username of the sender.
+        otp (str): The one-time password (OTP) for the new user.
```
Comment (zxenonx) [inline app/services/mail_service.py:99]: otp is not part of the function's params

**Review by zxenonx — COMMENTED** [review]
Good job @TolulopeJoel 
I left a comment

File: app/services/mail_service.py (hunk 1 above)
Comment (TolulopeJoel) [reply app/services/mail_service.py]: Fixed. Removed the OTP as parameter in the docstring

**Review by zxenonx — APPROVED** [review]
Thanks @TolulopeJoel

**Pull request merged**